department-of-veterans-affairs/caseflow
# Remove all references and logic for cached doc count on the front end

### Description
Removes all references and logic for cached document_count on the front end since the back end no longer supports it.

### Acceptance Criteria
- `cached` param is no longer passed to the back end

### Testing Plan
1. Go to anyone's queue
2. Ensure no calls to document_count include `?cached`
